### ISSUE TITLE
Let users choose to check for updates over Tor

### ISSUE THREAD

**Issue by holmesworcester** [TryQuiet/quiet#2582]
A colleague in the space was concerned when he saw that Quiet connected to S3, and wanted the option to disable this. 

Checking for updates exclusively over Tor seemed like the best option in this case, because we still want to check for updates and doing so over Tor would prevent us (or Amazon) from tracking the user's IP address (location) as it changes over time. 

If checking for updates and downloading updates over Tor works well we could make that the default behavior.

**Comment by Juul** [comment]
Yeah this breaks the threat model. Specifically the "NON-MEMBER cannot" section, given that Amazon and its employees are non-members. If there are privileged non-members then the threat model document needs to be updated.

**Comment by holmesworcester** [comment]
That is correct. I'll update it now.

**Comment by holmesworcester** [comment]
Updated: https://github.com/TryQuiet/quiet/wiki/Threat-Model

I tried to be as expansive as possible in describing who was an UPDATE PROVIDER and what they could do, but we can narrow this in the future and get more specific.
